File: FrontEnd/src/app/models/user.dto.ts
export interface UserDTO {
    id?: number;
    firstName: string;
    lastName: string;
    createdDate: string;
    updatedDate: string;
    email:string;
    role: string;
    profilePicURI?: string;
    resumeURI?: string;
    userName?:string;
    userOption?:string;
  }
  

File: BackEnd/src/main/java/com/example/futurbe/dto/userDTO/UserDTO.java
package com.example.futurbe.dto.userDTO;

import com.example.futurbe.entitys.Option;
import com.example.futurbe.entitys.Role;
import lombok.Data;
import java.util.Date;

@Data
public class UserDTO {
    private Long id;
    private String firstName;
    private String password;
    private String lastName;
    private Date createdDate;
    private Date updatedDate;
    private Option userOption;
    private  String username;
    private int LoginAttempts;

    private String email;

    private String profilePicURI;
    private String resumeURI;
    private Role role;
    private ResumeDTO resume;

}
